### PR TITLE
Feature/gh 11619 Create new component for error and warning messages

### LINKED ISSUE #11619: Create new component for CpqConfiguratorConflictAndErrorMessages
Create a new component for conflict and error messages.
Only add on configuration page, not on overview. 

Style similar to global messages (**NOT** sticky).

Both error and conflict messages should be displayed separately.
When more than one error/conflict message will be shown, group them together and make them collapsable via show/hide toogle. --> Discuss how the collapse text should be.

### PR DESCRIPTION
**Pull request by schuppell**

### CODE REVIEW

File: feature-libs/product-configurator/rulebased/components/configurator-conflict-and-error-messages/configurator-conflict-and-error-messages.component.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  showErrors = false;
+
+  toggleErrors(): void {
+    this.showErrors = !this.showErrors;
+  }
+
+  /* clear(type: GlobalMessageType, index: number): void {
+    this.globalMessageService.remove(type, index);
+  }*/
```
Comment (Uli-Tiger) [inline feature-libs/product-configurator/rulebased/components/configurator-conflict-and-error-messages/configurator-conflict-and-error-messages.component.ts]: can you plz. remove the commented out code

**Review by Uli-Tiger — CHANGES_REQUESTED** [review]
I assume most of the styling was copied from some other Spartacus component?
Is it possible to reuse the styling instead? - i see here the danger of additional maintenance effort for us, as we might need to copy the styling over again and again.

File: feature-libs/product-configurator/rulebased/components/configurator-conflict-and-error-messages/configurator-conflict-and-error-messages.component.ts (hunk 1 above)
Comment (schuppell) [reply feature-libs/product-configurator/rulebased/components/configurator-conflict-and-error-messages/configurator-conflict-and-error-messages.component.ts]: Not without changing the core css files.
So for now I would leave it like we currently have it.

Comment (Uli-Tiger) [reply feature-libs/product-configurator/rulebased/components/configurator-conflict-and-error-messages/configurator-conflict-and-error-messages.component.ts]: @schuppell
 
> Not without changing the core css files.
> So for now I would leave it like we currently have it.

is the reply to

>I assume most of the styling was copied from some other Spartacus component?
>Is it possible to reuse the styling instead? - i see here the danger of additional maintenance effort for us, as we might need >to copy the styling over again and again.

and not to 

>can you plz. remove the commented out code

Comment (Changsuwan) [reply feature-libs/product-configurator/rulebased/components/configurator-conflict-and-error-messages/configurator-conflict-and-error-messages.component.ts]: done

**Review by Uli-Tiger — APPROVED** [review]

**Pull request merged**